src/components/sky-nav/sky-nav.ts

```diff
@@ -74,34 +65,25 @@ export const initSkyNav = (navButton: HTMLButtonElement) => {
     const heightDiff = menu.getBoundingClientRect().height;
     if (isExpanded) {
       // Closing menu: slide the elements up before hiding the menu
-      for (const el of elementsToShift) {
-        el.style.transition = transition;
-        el.style.transform = `translateY(${-heightDiff}px)`;
-      }
+      document.body.style.transition = transition;
+      document.body.style.transform = `translateY(${-heightDiff}px)`;
 
       timeoutId = setTimeout(() => {
         menu.hidden = true;
-        for (const el of elementsToShift) {
-          el.style.transition = '';
-          el.style.transform = '';
-        }
+        document.body.style.transition = '';
+        document.body.style.transform = '';
```

Comment:
I realize this wasn't introduced in this PR, but is there any reason we're not using `style.setProperty` and `style.removeProperty`?

https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/removeProperty

Reply:
I might have put it in there in the first place but I have no preference between the two syntaxes.

Reply:
Updated: https://github.com/cloudfour/cloudfour.com-patterns/pull/1986/commits/84232b630079da353aaf078db86ee2da454c07ed